code/game/atoms.dm

```diff
@@ -442,6 +442,9 @@ var/list/blood_splatter_icons = list()
 /atom/proc/ratvar_act()
 	return
 
+/atom/proc/rcd_act(mob/user, var/obj/item/weapon/rcd/the_rcd)

```

Comment:
Well it can allow for even more RCD interactions(and probably less snowflake istypes!)